test/isTabbable.test.js

```diff
@@ -293,4 +293,96 @@ describe('isTabbable', () => {
       expect(isTabbable(getByTestId(container, 'radioB'))).toBe(false);
     });
   });
+
+  describe('display check', () => {
+    const fixture = `
+      <div data-testid="displayed-top" tabindex="0">
+        <div data-testid="displayed-nested" tabindex="0"></div>
+        <div
+          data-testid="displayed-zero-size"
+          tabindex="0"
+          style="width: 0; height: 0"
+          data-jsdom-no-size
+        ></div>
+      </div>
+      <div
+        data-testid="displayed-none-top"
+        tabindex="0"
+        style="display: none"
+        data-jsdom-no-size
+      >
+      <div data-testid="nested-under-displayed-none" tabindex="0" data-jsdom-no-size></div>
+    </div>
+    `;
+    it('return browser visible elements by default ("full" option)', () => {
+      const container = document.createElement('div');
+      container.innerHTML = fixture;
+      mockElementsSizes(container);
+      document.body.append(container);
+      const displayedTop = getByTestId(container, 'displayed-top');
+      const displayedNested = getByTestId(container, 'displayed-nested');
+      const displayedZeroSize = getByTestId(container, 'displayed-zero-size');
+      const displayedNoneTop = getByTestId(container, 'displayed-none-top');
+      const nestedUnderDisplayedNone = getByTestId(
+        container,
+        'nested-under-displayed-none'
+      );
```

Comment:
Same here: Can we move to a `beforeEach()` block to simplify the test code in here?

Reply:
unify under a setup function